### ISSUE TITLE
Add additional columns on the chapter admin page based on program info

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4775]
The data in the program info fields should be added as a few additional columns on the chapter admin page (/admin/chapters) so it can be exported:

- [x] Estimated Program Start (Question 5 - When do you estimate will be the start of your Technovation Girls program this season?)
- [x] Meeting Times (Question 8 - When do you plan to hold meetings for your participants? (multiple selection)During school, After school, In the evenings, On the weekend)
- [x] Program Length (Question 9 - How long are you planning to run your Technovation program? (single selection) Less than 6 weeks, 6-9 weeks, 10-12 weeks, 13-15 weeks, 16+ weeks)
- [x] Meeting Facilitator (Question 10 - Who will facilitate the meetings for your students? (multiple selection) Volunteer community mentors, Volunteer corporate mentors, Teachers, Paid staff, College students or faculty, Parents
- [x] Estimated Participation (Question 11 - How many participants do you anticipate serving with the program this year? (single selection) 20 - 50, 50 - 100, 100 - 200, 200 - 500, 500+)
- [x] Percent Underresourced (Question 12 - What percentage of the participants you plan to serve are low-income and/or under-resourced? 5%, 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 100%

**Comment by viviancan** [comment]
The listed columns are on the Chapter grid and on PREVIEW

**Comment by dboyer** [comment]
Tested on Preview and confirmed:

- Can add "program start date" as an additional column
- Can add "meeting start times" as an additional column
- Can add "program length" as an additional column
- Can add "estimated participation" as an additional column
- Can add "percent underresourced" as an additional column

I was able to export all columns.



@viviancan I don't know if it matters but the date is stored in a different format on the chapter info page vs the in the chapter grid. It's probably fine but just confirming that is okay from a technical perspective.
<img width="300" alt="Screen Shot 2024-06-25 at 9 58 04 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/d75fca6b-5482-4926-9bb4-d2dae30595ad">

<img width="270" alt="Screen Shot 2024-06-25 at 9 58 14 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/c64ead46-4934-40b1-872c-aed09eb1333e">
